### ISSUE TITLE
About storage space usage issues

### ISSUE THREAD

**Issue by xiangshibo** [chrislusf/seaweedfs#959]
My file shows only 410G in linux, but it takes up 530G when stored on seaweedfs. Is this space consumption normal, or what do I need to do?  thanks！

**Comment by chrislusf** [comment]
please show details.

**Comment by chrislusf** [comment]
updates?

**Comment by xiangshibo** [comment]
I use the [df -h] command to check the usage of the hard disk. My local files are placed under a directory. Looking at the occupancy of seaweed, I found that there are more than one hundred Gs than the actual space. I have saved some pdf files and jpg files. The file size is relatively small, and jpg averages only 300-400kb. I used a java client to upload. The maven configuration is as follows:
![image](https://user-images.githubusercontent.com/11767971/57991470-c3a34c00-7ae1-11e9-8539-9cd4910efd0b.png)
